Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift

```diff
@@ -64,11 +65,15 @@ class RubricCircleView: UIView {
             let font: UIFont
             let color: UIColor
             let bgColor: UIColor
+            let format = NSLocalizedString("g_points", bundle: .core, comment: "")
+            var a11yLabel: String = String.localizedStringWithFormat(format, Int(r))
+            a11yLabel += description
```

Comment:
I did all these (including line 76) separately as they each represent a piece of information I want VO to read and they aren't necessarily related.